examples/iam_v2/service_accounts_v2.yml

```
@@ -0,0 +1,189 @@
+---
+# Summary:
+# This playbook will do:
+# 1. Create service account
+# 2. Update service account
+# 3. List all users
+# 4. Fetch only service accounts using filter
+# 5. Fetch service account info using user ext_id criteria
+# 6. Create an API key
+# 7. List all API keys
+# 8. List all API keys with filter
+# 9. List all API keys using limit
+# 10. Get details of specific API key
+# 11. Create an authorization policy (ACP)
+# 12. Delete the ACP
+# 13. Revoke an API key
+# 14. Delete an API key
+
+- name: Service Accounts playbook
+  hosts: localhost
+  gather_facts: false
+  module_defaults:
+    group/nutanix.ncp.ntnx:
+      nutanix_host: <pc_ip>
+      nutanix_username: <user>
+      nutanix_password: <pass>
+      validate_certs: false
+
+  tasks:
+    - name: Setting Variables
+      ansible.builtin.set_fact:
+        username: test-ansible-service-account-1
+        email_id: test@test.com
+        email_id_updated: test_updated@test.com
+        service_acc_description: test-ansible-description-1
+        api_key_name: test-ansible-api-key-1
+        api_key_description: test-ansible-description-api-key-1
+
+    - name: Create Service Account
+      nutanix.ncp.ntnx_users_v2:
+        user_type: SERVICE_ACCOUNT
+        username: "{{ username }}"
+        email_id: "{{ email_id }}"
+        description: "{{ service_acc_description }}"
+      register: result
+      ignore_errors: true
```

Please remove ignore errors